pywbem_mock/_wbemconnection_mock.py

```diff
@@ -270,7 +270,7 @@ class FakedWBEMConnection(WBEMConnection, ResolverMixin):
     """
     def __init__(self, default_namespace=DEFAULT_NAMESPACE,
                  use_pull_operations=False, stats_enabled=False,
-                 timeout=None, response_delay=None):
+                 timeout=None, response_delay=None, verbose=False):
```

Comment:
Adding a "verbose" argument at an external interface goes beyond refactoring. Besides that, it is not used. I suggest to remove it.

Reply:
Removed.  Since I concluded that my previous approach to separating the repository data store from the CIMOM functions was too complex, I concluded that the API for the data store will be based on the current interface of dictionaries and once a CIMOM function (ex. get_class) validates that the namespace exists it can use dictionary methods on that repo.  There are still some function that do not do this but will clean them up.  Since most databases do support table to dictionary, this should work if the repository ever wants to use a database as the store.

Documented. that.  Next step is to move the repository to its own class.

Reply:
DONE